packages/manager/modules/octavia-load-balancer/src/create/routing.js

```diff
@@ -37,8 +38,57 @@ export default /* @ngInject */ ($stateProvider) => {
               return filtered;
             }, []),
           ),
+      regionsPlans: /* @ngInject */ ($http, $q, projectId, coreConfig) =>
+        $q
+          .all({
+            plans: $http.get(
+              `/cloud/project/${projectId}/capabilities/productAvailability?addonFamily=${AGORA_ADDON_FAMILY}&ovhSubsidiary=${
+                coreConfig.getUser().ovhSubsidiary
+              }`,
+            ),
+            privateNetworks: $http.get(
+              `/cloud/project/${projectId}/network/private`,
+            ),
+          })
+          .then(({ plans, privateNetworks }) =>
+            // Get only hourly plans and format them
+            plans.data.plans.reduce((filtered, plan) => {
+              const found = plan.code.match(SIZE_FLAVOUR_REGEX);
+              if (found) {
+                // If there is no private network for this region, we disable it
+                const mappedRegionPlans = plan.regions.map((region) => {
+                  let isRegionEnable = false;
+
+                  privateNetworks.data.forEach((privateNetwork) => {
+                    if (
+                      privateNetwork.regions.some(
+                        (privateNetworkRegion) =>
+                          privateNetworkRegion.region === region.name,
+                      )
+                    ) {
+                      isRegionEnable = true;
+                    }
+                  });
+
+                  return {
+                    ...region,
+                    hasEnoughQuota: () => isRegionEnable,
+                  };
+                });
+
+                filtered.push({
+                  code: found[1],
+                  regions: mappedRegionPlans,
+                  disabled: false,
+                });
+              }
+              return filtered;
+            }, []),
+          ),
```

Comment:
**suggestion(perf):** Once we find the hourly plan of Octavia load balancer, no need to continue to loop on Agora plans. So usage of `Array.prototype.find` is more appropriate than `Array.prototype.reduce`.
While evaluating the region availability, once we find a private-network in region, no need to continue the loop on private-networks as the region will be enabled. So usage of `Array.prototype.some` is more relevant than `Array.prototype.forEach`.
And I dont see the point of adding `code: found[1],` & `disabled: false,`. Since the variable name suggests `regionsPlans` this function must return array of regions only. 
```suggestion
            { 
            const octaviaHourlyPlan = plans.data.plans.find((plan) => plan.code.match(SIZE_FLAVOUR_REGEX));
            const mappedRegionPlans = octaviaHourlyPlan.regions.map((region) => {
            // If there is no private network for this region, we disable it
               const isRegionEnable = privateNetworks.data.some(({regions}) => 
                   regions.some(({region: privateNetworkRegion}) => privateNetworkRegion === region.ame)
               );
               return {
                  ...region,
                  hasEnoughQuota: () => isRegionEnable,
               };
            });
            return mappedRegionPlans;
            }),
```